cpu/driver/runtest.sh

```diff
@@ -25,6 +25,25 @@ TMPDIR=/var/tmp/$(date +"%Y%m%d%H%M%S")
 
 source $CDIR/../../cpu/common/libutil.sh
 
+NON_PSTATE_PROCESSORS="26 31 46"
+skip_pstate()
```

Comment:
Minor suggestion: do you think that maybe this could be clearer if it was called `check_pstate_support` instead of `skip_pstate` (and flip the 0 and the 1s in the `return`s, of course)? I'm thinking that, usually, shell stuff uses 0 for "success" and non-zero for failure, so I was already confused, wondering if this function would use that convention, or a bool-like convention (as it seems to be using).

On top of that, having a negative ("skip") instead of a positive ("support") added a layer of complexity for me, especially since I wasn't sure if unknown models would skip pstate or not (in contrast, it's more or less intuitive that you should not assume that something is supported if you can't even detect what it is).

That said, it's a minor thing so feel free to ignore if you prefer it the way it is now.

Reply:
Yes - I was using a bool like convention and I certainly didn't want 0 to be true.  That's probably more of a "C" thing.

I can make the changes you are suggesting.